test/test_parser.cpp

```diff
@@ -2318,6 +2333,81 @@ TEST_TYPES(Parser_list_of_primitive_types, Int, Optional<Int>, Bool, Optional<Bo
     }
 }
 
+TEST(Parser_list_of_primitive_mixed)
+{
+    Group g;
+    TableRef t = g.add_table("table");
+
+    constexpr bool nullable = true;
+    auto col_list = t->add_column_list(type_Mixed, "values", nullable);
+    CHECK_THROW_ANY(t->add_search_index(col_list));
+
+    Obj obj_empty_list = t->create_object();
+    Obj obj_with_null = t->create_object();
+    obj_with_null.get_list<Mixed>(col_list).add(Mixed{});
+    Obj obj_with_empty_string = t->create_object();
+    obj_with_empty_string.get_list<Mixed>(col_list).add(Mixed{""});
+    Obj obj_with_ints = t->create_object();
+    auto ints_list = obj_with_ints.get_list<Mixed>(col_list);
+    ints_list.add(Mixed{0});
+    ints_list.add(Mixed{1});
+    ints_list.add(Mixed{2});
+    Obj obj_with_numerics = t->create_object();
+    auto numeric_list = obj_with_numerics.get_list<Mixed>(col_list);
+    numeric_list.add(Mixed{1});
+    numeric_list.add(Mixed{Decimal128(2.2)});
+    numeric_list.add(Mixed{float(3.3f)});
+    numeric_list.add(Mixed{double(4.4)});
+    Obj obj_with_strings = t->create_object();
+    auto strings_list = obj_with_strings.get_list<Mixed>(col_list);
+    strings_list.add(Mixed{"one"});
+    strings_list.add(Mixed{"two"});
+    strings_list.add(Mixed{"three"});
+    strings_list.add(Mixed{""});
+    strings_list.add(Mixed{StringData{}});
+    Obj obj_with_mixed_types = t->create_object();
+    auto mixed_list = obj_with_mixed_types.get_list<Mixed>(col_list);
+    mixed_list.add(Mixed{"foo"});
+    mixed_list.add(Mixed{1});
+    mixed_list.add(Mixed{Timestamp(1, 0)});
+    mixed_list.add(Mixed{Decimal128(2.5)});
+    mixed_list.add(Mixed{float(3.7)});
+    mixed_list.add(Mixed{ObjectId::gen()});
+    mixed_list.add(Mixed{UUID()});
+    mixed_list.add(Mixed{});
+    mixed_list.add(Mixed{false});
+    mixed_list.add(Mixed{true});
+
+    verify_query(test_context, t, "values.@count == 0", 1);
```

Comment:
Can the aggregate functions only be invoked through a query, or can we use methods like `list.min()`, `list.max()`?

Reply:
Thanks for pointing this out, I've added support for that.